filesystem/src/test/java/com/nytimes/android/external/fs3/SourceDiskDaoStoreTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.nytimes.android.external.fs3
+
+
+import com.google.common.base.Charsets.UTF_8
+import com.google.gson.Gson
+import com.nytimes.android.external.store3.base.Fetcher
+import com.nytimes.android.external.store3.base.impl.BarCode
+import com.nytimes.android.external.store3.base.impl.StoreBuilder
+import com.nytimes.android.external.store3.middleware.GsonSourceParser
+import io.reactivex.Maybe
+import io.reactivex.Single
+import okio.BufferedSource
+import okio.Okio
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+import java.io.ByteArrayInputStream
+
+class SourceDiskDaoStoreTest {
+    @Mock
+    lateinit var fetcher: Fetcher<BufferedSource, BarCode>
+    @Mock
+    lateinit var diskDAO: SourcePersister
+    private val barCode = BarCode("value", KEY)
+
+    @Test
+    fun testSimple() {
```

Comment:
Nit: Can we rename this to something more descriptive?

Reply:
done!